vita3k/gui/src/settings_dialog.cpp

```diff
@@ -276,17 +276,17 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
             ImGui::Spacing();
             ImGui::Checkbox("Experimental: LLE libkernel & driver_us", &config.lle_kernel);
             if (ImGui::IsItemHovered())
-                ImGui::SetTooltip("Enable this for using libkernel and driver_us module (experimental).");
+                ImGui::SetTooltip("Enable this to use libkernel and driver_us modules (experimental).");
             ImGui::Spacing();
             if (ImGui::RadioButton("Automatic", config.auto_lle))
                 config.auto_lle = true;
             if (ImGui::IsItemHovered())
-                ImGui::SetTooltip("Select Automatic mode for using modules list pre-set.");
+                ImGui::SetTooltip("Select Automatic mode to select modules automatically from a pre-set list.");
```

Comment:
Not fan here, répétition of automatic word

Reply:
That's a bit odd, I agree. I reworded it.